src/Microsoft.AspNetCore.Mvc.WebApiCompatShim/Conventions/IUseWebApiActionConventions.cs

```diff
@@ -3,6 +3,9 @@
 
 namespace Microsoft.AspNetCore.Mvc.WebApiCompatShim
 {
+    /// <summary>
+    /// Indicates actions without attribute routes in a controller use WebAPI routing conventions.
```

Comment:
Oh actually scratch that. It's not entirely clear to me here if we're referring to the "web API" generic concept, or the specific product "ASP.NET Web API." Which is it?